src/filters.js

```diff
@@ -119,10 +119,12 @@ var filters = {
         if(str instanceof r.SafeString) {
             return str;
         }
+        str = (str === null || str === undefined) ? '' : str;
```

Comment:
Could you please fix normalize function and remove false from there and use normalize here.
I don't see any idea behind it